examples/google-signin/google-signin-iframe.html

```diff
@@ -43,10 +47,10 @@
     };
 
     window.onload = () => {
-        const creator = new SwgGoogleSigninCreator(
-          ALLOWED_ORIGINS, CLIENT_ID, publisherGoogleSignInCallback, window
```

Comment:
Could you inform Martin about this change?

Reply:
For sure